lib/xlsxtream/xml.rb

```diff
@@ -36,8 +39,18 @@ def escape_attr(string)
         string.gsub(UNSAFE_ATTR_CHARS, XML_ESCAPES)
       end
 
+      # Add underscore to strings that merely look like hex values, preventing manipulation into invalid UTF8
+      # Per Microsoft Open Specifications for Excel:
+      # Underscore (0x005f): This character shall be escaped only when used to escape the first underscore character in the format _xHHHH_.
+      def encode_underscores_using_x005f(string)
```

Comment:
🔧 function name suggestion: `ensure_strings_that_match_escape_sequence_are_escaped`
a bit wordy, but I feel that it concisely captures the intent of the method

Reply:
What about `escape_strings_that_match_escape_sequence`?